securedrop/journalist_templates/col.html

```diff
@@ -19,10 +19,10 @@ <h1 class="visually-hidden">{{ source.journalist_designation }}</h1>
   <form action="/bulk" method="post">
     <div>
       <div id="select-container"></div>
-      <button type="submit" name="action" value="download" class="small icon">
+      <button aria-label="{{ gettext('Download Selected Files') }}" type="submit" name="action" value="download" class="small icon">
         {{ gettext('Download Selected') }}
       </button>
-      <a href="#delete-selected-confirmation-modal" id="delete-selected-link" class="btn small danger">
+      <a aria-label="{{ gettext('Delete Selected Files') }}" href="#delete-selected-confirmation-modal" id="delete-selected-link" class="btn small danger">
         {{ gettext('Delete Selected') }}
       </a>
```

Comment:
These make such a difference; thank you!

Reply:
Happy to work on it, hopefully folks will find these changes helpful and meaningful!